routes/web.php

```diff
@@ -19,14 +17,23 @@
 //Route::get('/', function () {
   //  return view('welcome');
 //});
-
+Route::redirect('/', '/homepage');
 Route::get('/homepage', function(){return view('homepage');})->name('homepage');
-Route::get('/booklist', function(){return view('booklist');})->name('booklist');
+Route::get("/books", [BookController::class,'show'])->name('books');
```

Comment:
Semih'in attığı convention'a bakarak buradaki fonksiyon ismini "show" yerine "index" yapsak daha iyi olur.